survival_guide/3_operations.adoc

```diff
@@ -590,32 +590,56 @@ Our recommended configuration for using the Spark History Server with Fusion is
 [source,yaml]
 ----
 gcs:
-    enableGCS: false
-    secret: history-secrets
-    key: [SECRET_KEY_NAME].json
-    logDirectory: gs://[BUCKET_NAME]
+  enableGCS: true
+  secret: history-secrets
+  key: sparkhistory.json
+  logDirectory: gs://[BUCKET_NAME]
 service:
-    type: ClusterIP
-    port: 18080
+  type: ClusterIP
+  port: 18080
+
+nfs.enableExampleNFS: false
```

Comment:
for some reason this doesn't actually disable the NFS pod! You need to use a different notation.
```suggestion
nfs:
  enableExampleNFS: false
```

Reply:
Sorry, you also need to add another thing or pod won't schedule
```suggestion
pvc:
  enablePVC: false
nfs:
  enableExampleNFS: false
```